hummingbot/strategy/arbitrage/arbitrage.pyx

```diff
@@ -469,15 +439,15 @@ cdef class ArbitrageStrategy(StrategyBase):
             buy_fee = buy_market.c_get_fee(
                 buy_market_trading_pair_tuple.base_asset,
                 buy_market_trading_pair_tuple.quote_asset,
-                OrderType.MARKET,
+                OrderType.LIMIT,
```

Comment:
This is another confusion, for the `c_get_fee` we want to get `taker` fee but to do that we'll have to pass in MARKET order type.

Reply:
I understand this. I'll modify now.

Reply:
@vic-en I think I mentioned earlier about fixing `c_get_fee` to take correct parameters, I think we should leave it as it is for now (as fixing it would be quite complicated, it involves fix `hummingsim` as well. 
So here you can either call `c_get_fee` by passing in LIMIT order type to get maker fee or MARKET to get taker fee, or you can simply call your `estimate_fee` module directly.

Reply:
Sorry, I forgot to do that. I pushed a commit to address that just now. Kindly review again.